README.md

```diff
@@ -4,26 +4,26 @@
 
 # Snowplow Enrich
 
-Snowplow Enrich is a set of applications and libraries for processing raw Snowplow events into validated and enriched Snowplow events, ready for loading into [Storage][storage].
+Snowplow Enrich is a set of applications and libraries for processing raw Snowplow events into validated and enriched Snowplow events.
 It consists of following modules:
 
 * Snowplow Common Enrich - a core library, containing all validation and transformation logic. Published on Maven Central
 * Snowplow Stream Enrich - a set of applications working with Kinesis, Kafka and NSQ. Each asset published as Docker image on DockerHub
 * Snowplow Beam Enrich - a Google Dataflow job. Published as Docker image on DockerHub
 * Snowplow FS2 Enrich - an application for a GCP pipeline that does not require a distributed computing framework. Published as Docker image on DockerHub
 
-Snowplow Enrich provides record-level enrichment only: feeding in 1 raw Snowplow event will yield 0 or 1 records out, where a record may be an enriched Snowplow event or a reported bad record.
+Snowplow Enrich provides record-level enrichment only: feeding in 1 raw Snowplow event will yield exactly 1 record out, where a record may be an enriched Snowplow event or a reported bad record.
```

Comment:
Not exactly. A raw `CollectorPayload` can for instance contain 3 events where 2 would go out as enriched event and one as bad row.